setup-dev.sh

```diff
@@ -5,6 +5,24 @@ echo "# Install testing tools."
 echo "# Only works with Ubuntu / APT."
 echo "# --------------------------------------"
 
+hookScriptName=".git/hooks/pre-push"
+
+if [ -t 1 ]; then
+    echo "Install a git pre-push hook?"
```

Comment:
Let's make sure it's clear what this hook is going to do: "Install a git hook to automatically lint files before pushing? (y/N) "